src/style.css

```diff
@@ -1 +1,29 @@
-/* add styles here */
+body {
+  margin: 0;
+}
+
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  display: flex;
+  background-image: url(../src/images/star.svg);
+  background-position: center;
+  height: 16px;
+  width: 16px;
+  background-repeat: no-repeat;
+  margin-left: 4px;
+}
```

Comment:
Don't use tag names for styling (except `html` and `body`). Instead of styling `.stars__star`, consider adding a class to each star and style that class. This will make your styles more modular and avoid potential conflicts if you have other elements with the same tag name in different contexts.